### PR TITLE
Add governance and organizational structure information

### LINKED ISSUE #3: Onboarding Guide Request
### Please describe the resource requested

A onboarding guide that explains the organizational structure, how to join, the meaning of different membership levels, and an overview of how you can be involved would be exceptionally helpful. 

### Why would you like it added

So that folks joining the CDSC project know what is going on and how they can be involved. What is the legal structure and why was it chosen? Is it even called the CDSC or is it now LFESS? 

### How soon do you need this added

As new people come on board and want to contribute.

### Additional context

There's been so much change in the organizational structure over different iterations that I've lost track of the current organizational structure.

### PR DESCRIPTION
**Pull request by jmertic**

### CODE REVIEW

**Comment by jmertic** [comment]
@jorydotcom would you mind doing a review?

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+## Participating and contributing
+
+The working groups are open to anyone, and there are no financial requirements, or corporate affiliations needs for participation. Participation includes signing up for mailing lists, joining Slack channels, or participating in a working group or community meeting. These groups are open and transparent; each group defines how to participate in their individual working group repositories.
+
+If someone wants to contribute to a specification, then a [membership](#membership) is required. Contributing to a specification is defined as anything said or written by a person that would be incorporated into a specification. Examples could include:
+
+- A new issue or pull request opened requesting a change to a specification
+- Comments on an issue or pull request that is making a change to a specification.
+- Comments shared during a meeting that are directly incorporated into a specification.
+
```
Comment (Henry-WattTime) [inline GOVERNANCE.md:66]: "Information on how to participate in the working groups can be found here<link>"

Comment (Henry-WattTime) [reply GOVERNANCE.md:66]: Would this be helpful? can we link to the respective projects?

Comment (jmertic) [reply GOVERNANCE.md:66]: I like the idea of a seperate document that catalogs all the projects; let me add that.

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+# LFESS Governance
+
+LF Energy Standards and Specifications (LFESS) is structured as a Series LLC entity underneath the Joint Development Foundation Projects, LLC entity, a similar legal entity structure used for technical projects hosted at the LF Energy Foundation. Please refer
+
+## Governance structure
+
+The below chart describes the governance structure.
+
+```mermaid
```
Comment (Henry-WattTime) [inline GOVERNANCE.md:9]: can the text in this chart link out to the different committees and working groups?

Comment (jmertic) [reply GOVERNANCE.md:9]: That I'm not sure of - need to play more with mermaid.

Comment (jmertic) [reply GOVERNANCE.md:9]: Figured it out - check now.

**Pull request merged**